kbase-extension/static/kbase/js/widgets/appWidgets2/input/intInput.js

```diff
@@ -153,45 +159,47 @@ define([
             // CONTROL
             const min = spec.data.constraints.min,
                 max = spec.data.constraints.max;
-            let initialControlValue;
-            if (typeof currentValue === 'number') {
-                initialControlValue = String(currentValue);
+
+            function boundaryDiv(value, isMin) {
```

Comment:
See above in floatInput.js - this is duplicate code there and will get moved in the next PR.